### PR TITLE
Il Pianto Della Luna

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds Il Pianto Della Luna.

## Why It's Good For The Game
Hot Single moms in your area

## Changelog
:cl:
add: Added Il Pianto della Luna
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/waw.dm [hunk 1]
```diff
@@ -744,12 +744,60 @@
...
+		return
+	inuse = TRUE
+	if(do_after(user, 30))	//3 seconds for a big heal.
+		playsound(src, 'sound/magic/staff_healing.ogg', 200, FALSE, 9)
+		for(var/mob/living/carbon/human/L in range(5, get_turf(user)))
+			L.adjustSanityLoss(10)
+	inuse = FALSE
+
+
+/obj/item/ego_weapon/heaven
```
Comment (dsmith328) [inline code/game/objects/items/ego_weapons/waw.dm:775]: Why is THIS here?

File: code/modules/mob/living/simple_animal/abnormality/waw/luna.dm [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+		for(var/mob/living/carbon/human/L in GLOB.player_list)
+			L.apply_status_effect(STATUS_EFFECT_LUNAR)
+
+		if(!performance)
+			performance = TRUE
+			playsound(user, 'sound/abnormalities/luna/moonlight_sonata.ogg', 100, FALSE, 28)	//Let the people know.
+
+		if(breached)	//You will have to start a new performance to delete the breached abno.
+			killspawn = TRUE
+	return TRUE
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/waw/luna.dm:82]: Specifically could they walk off to short-stop the downside of the Performance? If so, should there be a larger punishment for doing so?

File: code/modules/mob/living/simple_animal/abnormality/waw/luna.dm [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+/mob/living/simple_animal/hostile/luna/proc/AOE()
+	for(var/turf/T in view(aoerange, src))
+		new /obj/effect/temp_visual/revenant(T)
+		for(var/mob/living/L in T)
+			if(faction_check_mob(L, FALSE))
+				continue
+			if(L.stat == DEAD)
+				continue
+			L.apply_damage(aoedamage, BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+	aoeactive = FALSE
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/waw/luna.dm:153]: Her melee is good but this AoE might be a little underwhelming for WAW.

File: code/modules/mob/living/simple_animal/abnormality/waw/luna.dm [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+//	gift_type =  /datum/ego_gifts/moonlight
+	var/performance = FALSE
+	var/performance_length = 60 SECONDS
+	var/breach_length = 404 SECONDS		//How long the song is (when I finally finish it)
+	var/breached = FALSE
+	var/breached_monster
+	var/killspawn
+
+/mob/living/simple_animal/hostile/abnormality/luna/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(50))
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/waw/luna.dm:37]: "Normal" probability is 40% if we compare to Queen Bee.

File: code/modules/paperwork/records/info/waw.dm [hunk 5]
```diff
@@ -514,3 +514,32 @@
...
+	Max PE Boxes : 20	<br>
+	Qliphoth Counter : 3	<br>
+	Work Damage Type : Black	<br>
+	Work Damage : High	<br>
+	- When the work result was Normal, the Qliphoth Counter lowered with a normal probability.	<br>
+	- When the work result was Bad, the Qliphoth Counter lowered.	<br>
+	- The piano can be played by ordering an Agent to Perform. However, please note that the Qliphoth Counter will lower when the performance is finished.	<br>
+	- The music temporarily increased the Temperance and Justice of all employees by a small degree. However, the one who performed lost half their HP by the end of the performance.	<br>
+	- The Qliphoth Counter lowered to 0 when an employee entered the unit to play for the third time.	<br>
+	- When Il Pianto della Luna escaped, it voluntarily returned to its Containment Unit after the end of the third movement. The movement also prematurely ended when Il Pianto della Luna was suppressed.	<br>
```
Comment (dsmith328) [inline code/modules/paperwork/records/info/waw.dm:533]: "after the end of the third movement" might be inaccurate to how it's coded. They must perform ANOTHER performance to get it to return, no?

**Review by dsmith328 — COMMENTED** [review]
Sprites look good, I'd like to see it in action.

File: code/modules/paperwork/records/info/waw.dm (hunk 5 above)
Comment (Kitsunemitsu) [reply code/modules/paperwork/records/info/waw.dm:533]: 3rd movement is when the song ends (after 404 seconds). But the song was long and I couldn't figure out how to end it prematurely.

File: code/game/objects/items/ego_weapons/waw.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/waw.dm:775]: Just some bonus, I had the sprites, and wanted to make use of it. 
It doesn't affect anything, gameplay wise.

File: code/modules/mob/living/simple_animal/abnormality/waw/luna.dm (hunk 3 above)
Comment (dsmith328) [reply code/modules/mob/living/simple_animal/abnormality/waw/luna.dm:153]: It is not

File: code/modules/mob/living/simple_animal/abnormality/waw/luna.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/abnormality/waw/luna.dm:82]: Nah, still fires due to work complete.

**Review by dsmith328 — APPROVED** [review]
I've seen it in action, looks good. Very scary. Update when custom song is gotten? I think.
